### PR TITLE
Add io.github.trackballs.Trackballs

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/trackballs/trackballs/issues/30
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/trackballs/trackballs/pull/29

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build io.github.trackballs.Trackballs

File: io.github.trackballs.Trackballs.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - --socket=pulseaudio
+command: trackballs
+rename-appdata-file: trackballs.appdata.xml
+rename-desktop-file: trackballs.desktop
+rename-icon: trackballs
+
+modules:
+  - libatomic_ops-7.6.10.json
+  - bdwgc-7.6.6.json
+  - guile-2.2.4.json
```
Comment (barthalion) [inline io.github.trackballs.Trackballs.yaml:20]: Please merge them into the main manifest.

File: io.github.trackballs.Trackballs.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+app-id: io.github.trackballs.Trackballs
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
```
Comment (barthalion) [inline io.github.trackballs.Trackballs.yaml:5]: Misses `--persist=.trackballs`.

**Comment by Mailaender** [comment]
bot, build io.github.trackballs.Trackballs

**Pull request closed without merging**